src/Resque/Helpers/Util.php

```diff
@@ -106,7 +108,10 @@ public static function human_time_diff($from, $to = null)
      * @param   mixed   key path string (delimiter separated) or array of keys
      * @param   mixed   value that was found
      * @param   string  key path delimiter
-     * @return  bool
+     * @param  mixed $array
```

Comment:
Here the parameter documentation doesn't match anymore with the function signature - maybe a merge artifact?

Reply:
shouldn't the `mixed $array` be indented one more space to the right after the `@param`?

Reply:
Oh, right the docblocks should be aligned as well after they are fixed here. Thanks :)

Reply:
autofix artifact, done